cmd/migrate/main.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2020 Google LLC
```

Comment:
It's a bit of code, but this lets us build migrations into a container, same as the verification server.